channels/routing.py

```diff
@@ -67,7 +68,32 @@ class URLRouter:
     _path_routing = True
 
     def __init__(self, routes):
-        self.routes = routes
+        new_routes = []
+        for route in routes:
+            if not route.callback and isinstance(route, URLResolver):
+                for url_pattern in route.url_patterns:
+                    url_pattern: URLPattern
```

Comment:
Can you remove this line? We don't use type-hints in channels (yet)

Reply:
OK, I've removed it